app/views/pages/privacy.html.erb

```diff
@@ -39,17 +37,16 @@
       </p>
 
       <h2 class="govuk-heading-l">The right to lodge a complaint</h2>
-
-      <p class="govuk-body">You have the right to raise any concerns with the Information Commissioner’s Office (ICO) via their website at <a href="https://ico.org.uk/concerns/" class="govuk-link">https://ico.org.uk/concerns/</a>.
+      <p class="govuk-body">
+        You have the right to raise any concerns with the Information Commissioner’s Office (ICO) via their website at
+        <%= govuk_link_to('https://ico.org.uk/concerns/', 'https://ico.org.uk/concerns/') %>.
```

Comment:
This URL seems to have been redirected...

```suggestion
        <%= govuk_link_to('https://ico.org.uk/make-a-complaint', 'https://ico.org.uk/make-a-complaint') %>.
```

Reply:
Nice spot! Updated.